src/fragments/lib/auth/ios/switch-auth/switch-auth.mdx

```diff
@@ -0,0 +1,84 @@
+
+For client side authentication there are three different flows:
```

Comment:
I actually had a different PR for this - https://github.com/aws-amplify/docs/pull/4739 . But yours have more information, I think I should close my PR.  Can you check if you can address Tim's comment in this PR as well.